src/main/java/org/opensearch/neuralsearch/processor/RRFProcessor.java

```diff
@@ -70,6 +72,7 @@ <Result extends SearchPhaseResult> void hybridizeScores(
         Optional<FetchSearchResult> fetchSearchResult = getFetchSearchResults(searchPhaseResult);
         boolean explain = Objects.nonNull(searchPhaseContext.getRequest().source().explain())
             && searchPhaseContext.getRequest().source().explain();
+        EventStatsManager.increment(EventStatName.RRF_PROCESSOR_EXECUTIONS);
```

Comment:
for rank based normalization you also need to increment counter for combination technique calls, currently we do support "rrf" https://docs.opensearch.org/docs/latest/search-plugins/search-pipelines/score-ranker-processor/

Reply:
no in near future, my worry was more about consistent result format, because for RRF we have "rrf" as both normalization technique (hidden), and combination technique.